com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ Additional documentation and release notes are available at [Multiplayer Documen
 
 ### Added
 
+- Added `NetworkManager.RemoveNetworkPrefab(...)` to remove a prefab from the prefabs list (#1950)
```

Comment:
just a note that this PR will also be considered for backport to `release/1.0.0` for `1.0.0-pre.10`